components/config/opts.rs

```diff
@@ -1030,7 +1035,7 @@ pub fn from_cmdline_args(opts: getopts::Options, args: &[String]) -> getopts::Re
         convert_mouse_to_touch: debug_options.convert_mouse_to_touch,
         exit_after_load: opt_match.opt_present("x"),
         no_native_titlebar: do_not_use_native_titlebar,
-        enable_vsync: !debug_options.disable_vsync,
+        enable_vsync: !disable_vsync(),
```

Comment:
Since `enable_vsync` is only used in `ports/glutin/headed_window.rs`, we don't actually need this member of Opts with the related changes.